config/install/gesso.settings.yml

```diff
@@ -0,0 +1,13 @@
+features:
+  node_user_picture: false
+  comment_user_picture: true
+  comment_user_verification: true
+  favicon: false
+logo:
+  use_default: false
+favicon:
+  use_default: true
+include_back_to_top: 0
```

Comment:
Wondering if this should default to on rather than off. Do you think people will still know it's there for use if it's off by default?

Reply:
Good catch! I meant to match the default set in `theme-settings.php`. I’ll make that change now.